supervisor/addons/model.py

```diff
@@ -539,13 +539,17 @@ def _available(self, config) -> bool:
             return False
 
         # Home Assistant
-        version = config.get(ATTR_HOMEASSISTANT) or self.sys_homeassistant.version
-        if pkg_version.parse(self.sys_homeassistant.version) < pkg_version.parse(
-            version
-        ):
-            return False
-
-        return True
+        version = config.get(ATTR_HOMEASSISTANT)
+        if version is None:
+            return True
+
+        try:
+            assert self.sys_homeassistant.version is not None
```

Comment:
What will happen if that is None, do that don't raise an InvalidVersion error?

Reply:
Yes, it will, it will throw a TypeError. I went to much with Paulus his suggestion on asserting there.

Fixed now.